src/test/regress/expected/multi_behavioral_analytics_basics.out

```diff
@@ -522,3 +522,210 @@ SELECT count(*), count(DISTINCT user_id), avg(user_id), avg(value_1_agg) FROM ag
      6 |     6 | 3.5000000000000000 | 0.16666666666666666667
 (1 row)
 
+-- test order by when pushing down avg
+SELECT user_id, avg(value_1)
```

Comment:
Minor: this test source is mostly intended for pushdownable `INSERT ... SELECT` queries. It'd be good if we can move them to somewhere else such as `multi_complex_expressions.sql`?

Reply:
done